kyco/napps_manager.py

```diff
@@ -0,0 +1,25 @@
+"""Manage Network Application files."""
+from os import listdir, path
+
+
+class NAppsManager:
+    """Deal with NApps at filesystem level."""
+
+    def __init__(self, options):
```

Comment:
Why not passing the 'folder' itself instead of the full options? [This would allow us to instantiate multiple Managers and/or avoid the need of instantiation of the KycoConfig class.